src/main/java/frc/robot/RobotContainer.java

```diff
@@ -32,18 +37,42 @@ public class RobotContainer {
   // The robot's subsystems
   private ADIS16470_IMU m_gyro = new ADIS16470_IMU();
   private final DriveSubsystem m_robotDrive = new DriveSubsystem(m_gyro);
-  private final Intake intake = new Intake();
```

Comment:
I disabled arm and intake because we were getting errors based on those can connections being gone.  Please put them back.

Reply:
This still need to be added back

Reply:
Added all the arm code back.